Libs/MRML/Widgets/Testing/qMRMLNodeComboBoxTest8.cxx

```diff
@@ -225,18 +226,17 @@ int qMRMLNodeComboBoxTest8(int argc, char* argv[])
   }
 
   // add a custom actions
-  const QStringList actionPrefixes = {
-    "Create new ",
-    "Delete current ",
-    "Edit current ",
-    "Rename current ",
-  };
-  for (const QString& actionPrefix : actionPrefixes)
+  const QStringList actionPrefixesData = { "Create new |CREATE", "Delete current |DELETE", "Edit current |EDIT", "Rename current |RENAME" };
+  for (const QString& actionPrefix : actionPrefixesData)
```

Comment:
I'll fix this by the end of the day.

Reply:
I just addressed the error and updated the topic :rocket: